From d67dccc128046b75e5dcb9c18616a908560a8711 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Thu, 26 Nov 2020 15:47:23 +0800
Subject: [PATCH] phy: rockchip: naneng-combphy: add pipe clk for rk3568

The pipe clk of rk3568 is used for PIPE_GRF module, we
need to manage this clk so that the USB IP can access
the registers of PIPE_GRF.

Change-Id: I9ebfd5d25bdf1e95a9c5a3390b1030aed5f5c7ba
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-naneng-combphy.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/phy/rockchip/phy-rockchip-naneng-combphy.c b/drivers/phy/rockchip/phy-rockchip-naneng-combphy.c
index 0944e2f66fd4..06c4bda4814d 100644
--- a/drivers/phy/rockchip/phy-rockchip-naneng-combphy.c
+++ b/drivers/phy/rockchip/phy-rockchip-naneng-combphy.c
@@ -428,6 +428,7 @@ static const struct rockchip_combphy_grfcfg rk3568_combphy_grfcfgs = {
 static const struct clk_bulk_data rk3568_clks[] = {
 	{ .id = "refclk" },
 	{ .id = "apbclk" },
+	{ .id = "pipe_clk" },
 };
 
 static const struct rockchip_combphy_cfg rk3568_combphy_cfgs = {
-- 
2.35.3

